google-cloud-tools-plugin/google-cloud-core/resources/config.properties

```diff
@@ -17,6 +17,7 @@
 # Place cloud tools plugin configuration here.
 feature.appengine.flex=true
 feature.managed.sdk=true
+feature.managed.sdk.update=false
```

Comment:
just to refresh my memory:

we can't release the managed SDK feature until the update is fully complete right? I guess if we wanted to, we could hide the automatic update UI element behind this flag too? No need to do that now though, just wondering.

Reply:
Sure, we can hide the UI as well - but the final step (mutual install / use blocking) is ready, and we can review it right after we're done with this PR, so we should be good.

Reply:
even better!